### PR TITLE
fix(generator-pie-component): DSW-000 export new native playwright config

### PR DESCRIPTION
**Pull request by siggerzz**
## Describe your changes (can list changeset entries if preferable)
---
"@justeattakeaway/generator-pie-component": minor
---

[Added] - New exports for native playwright configs

@justeattakeaway/pie-chip / @justeattakeaway/pie-cookie-banner
- Updated to new exported configs

Note: I've manually tested the generator with the new changes, and the component tests work as expected

## Author Checklist (complete before requesting a review, do not delete any)
- [x] I have performed a self-review of my code.
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.

## Not-applicable Checklist items
Please move any Author checklist items that do not apply to this pull request here.
- [-] I have reviewed visual test updates properly before approving.
- [-] I have added thorough tests where applicable (unit / component / visual).
- [-] If changes will affect consumers of the package, I have created a changeset entry.
- [-] If a changeset file has been created, I have tested these changes in [PIE Aperture](https://github.com/justeattakeaway/pie-aperture/) using the `/test-aperture` command.

---

## Testing
[How do I test my changes?](https://github.com/justeattakeaway/pie/wiki/PIE-Aperture)

| Task                   | Link                             |
|------------------------|----------------------------------|
| Aperture PR            | [🔗](#) |
| NextJS 14 deployment   | [🔗](#) |
| Nuxt 3 deployment      | [🔗](#) |
| Vanilla deployment     | [🔗](#) |

## Reviewer checklists (complete before approving)
Mark items as `[-] N/A` if not applicable.

### Reviewer 1
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
- [-] I have verified that all acceptance criteria for this ticket have been completed.
- [-] I have reviewed the Aperture changes (if added)
- [-] If there are visual test updates, I have reviewed them.

### Reviewer 2 - @fernandofranca 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview.
- [-] I have verified that all acceptance criteria for this ticket have been completed.
- [-] I have reviewed the Aperture changes (if added)
- [-] If there are visual test updates, I have reviewed them.

### CODE REVIEW

File: packages/components/pie-chip/playwright-lit-visual.config.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { defineConfig } from '@sand4rt/experimental-ct-web';
-import { getPlaywrightVisualConfig } from '@justeattakeaway/pie-components-config';
+import { defineConfig } from '@playwright/test';
```
Comment (siggerzz) [inline packages/components/pie-chip/playwright-lit-visual.config.ts:1]: >[!NOTE]
>pie-chip component / a11y tests haven't been refactored to native @playwright/test yet, hence why this is the only file that's been updated.

**Review by JoshuaNg2332 — APPROVED** [review]

File: configs/pie-components-config/playwright-native-visual-config.js
Comment (fernandofranca) [inline configs/pie-components-config/playwright-native-visual-config.js:1]: I see lots of similar values in `playwright-native-config.js`. 
Can we have a file with the common values, that gets consumed by both configs?
Then if we need to update any of the values, we do it once without risking to introduce a inconsistency.

Comment (siggerzz) [reply configs/pie-components-config/playwright-native-visual-config.js:1]: Happy to do this, but do you mind if I create a seperate ticket for this, currently this is blocking a couple of PR's 👍🏼

File: configs/pie-components-config/playwright-native-config.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        snapshotDir: './__snapshots__',
+        /* Maximum time one test can run for. */
+        timeout: 10 * 1000,
+        testIgnore: '*-react.spec.js',
+        /* Run tests in files in parallel */
+        fullyParallel: true,
+        /* Fail the build on CI if you accidentally left test.only in the source code. */
+        forbidOnly: !!process.env.CI,
+        /* Retry on CI only */
+        retries: process.env.CI ? 2 : 0,
```
[the remark this answers is not in the document]
Comment (siggerzz) [reply configs/pie-components-config/playwright-native-config.js:18]: This is just copy-pasted from the generated boilerplate config when you create a new playwright suite. Happy to re-evaluate our configs though as part of the ticket to share common properties 👍🏼

**Review by fernandofranca — APPROVED** [review]

File: configs/pie-components-config/playwright-native-visual-config.js
Comment (fernandofranca) [reply configs/pie-components-config/playwright-native-visual-config.js:1]: Sure!

Comment (siggerzz) [reply configs/pie-components-config/playwright-native-visual-config.js:1]: Created as part of DSW-2569 👍🏼

**Review by maledr5 — APPROVED** [review]

**Pull request merged**